src/lib/Sympa/List.pm

```diff
@@ -4165,10 +4165,6 @@ sub sync_include {
 
     $role ||= 'member';    # Compat.<=6.2.54
 
-    return 0
-        unless $self->has_data_sources($role)
-        or $self->has_included_users($role);
-
```

Comment:
I think this paragraph means that: Updating inclusion will be performed when:
- The list has any data sources, or
- The list has users who have been included, even if the list already doesn't have any data sources.

So it need not be removed for me.